### PR TITLE
verify missing condition in dao

### PR DESCRIPTION
**Pull request by vuvoth**
## Proposed Changes

 -  Add rekey_to verify in deposit_vote_token
 -  Add fee verifition in daoFundLsig. 
 - Tests
## TODO

- [ ] update change log

### CODE REVIEW

File: examples/dao/assets/dao-app-approval.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from ast import Global
```
Comment (vuvoth) [inline examples/dao/assets/dao-app-approval.py:1]:
```suggestion
```

File: examples/dao/assets/dao-fund-lsig.py [hunk 2]
```diff
@@ -23,20 +23,21 @@ def basic_checks(txn: Txn): return And(
...
+        Gtxn[1].fee() == Int(0)
     )
 
     # Opt-in transaction is allowed
     opt_in = And(
         basic_checks(Txn),
         Txn.type_enum() == TxnType.AssetTransfer,
-        Txn.asset_amount() == Int(0)
+        Txn.asset_amount() == Int(0),
+        Txn.fee() == Int(1000)
```
Comment (robert-zaremba) [inline examples/dao/assets/dao-fund-lsig.py:40]:
```suggestion
        Txn.fee() <= Int(2000)
```

File: examples/dao/assets/dao-app-approval.py [hunk 3]
```diff
@@ -28,7 +28,8 @@ def basic_checks(txn: Txn): return And(
     # * receiver (receiver of gov_token)
     def verify_deposit(tx_index: Int, receiver: Addr):
         return Assert(And(
-            Global.group_size() >= tx_index,
+            Global.group_size() > tx_index,
```
Comment (robert-zaremba) [inline examples/dao/assets/dao-app-approval.py:31]: wow, how possible we missed it?

**Review by robert-zaremba — APPROVED** [review]
Thanks for looking at it! Left few suggestions

**Pull request merged**